assignments/pr_review_tutorial/ingram/README.md

```diff
@@ -1,11 +1,10 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
+Ok so I guess you are reading this cuz you wanna use my code. This code imports factorial from ``simple_functions``
 
     >>> from simple_functions import factorial
     >>> factorial(10)
     9
 
-and this other part does something.  I forget why that I did it:
+This creates a mx via 
```

Comment:
Though it may be my own unfamiliarity with coding jargon, the use of "mx" has left me befuddled. Consider changing this.

Reply:
Mx is shorthand for matrix.